CHANGELOG.md

```diff
@@ -14,6 +14,32 @@ Looking for changes that affect our C API? See the [C API Changelog](lib/c-api/C
 
 ## Fixed
 
+## 3.0.0-rc.1 - 2022/10/25
+
+## Added
+
+- [#3215](https://github.com/wasmerio/wasmer/pull/#3215)  Update wasmer --version logic, integrate wapm-cli
+- [#3218](https://github.com/wasmerio/wasmer/pull/#3218)  Seal `HostFunctionKind`
+- [#3222](https://github.com/wasmerio/wasmer/pull/#3222)  Add function to retrieve function name from wasm_frame_t
+
+## Changed
+
+- [#3248](https://github.com/wasmerio/wasmer/pull/#3248)  Move loupe CHANGELOG entry from 2.3.0 to 3.x
```

Comment:
I think these links are broken because https://github.com/wasmerio/wasmer/pull/#3248 takes you to the main pull request page.

We probably want https://github.com/wasmerio/wasmer/pull/3248